autoinstrumentation/nginx/Dockerfile

```diff
@@ -0,0 +1,25 @@
+
+############################
+# STEP 1 download the webserver agent 
+############################
+FROM alpine:latest as agent
+
+ARG version
+
+RUN mkdir /opt/opentelemetry
+WORKDIR /opt/opentelemetry
+
+RUN wget https://github.com/open-telemetry/opentelemetry-cpp-contrib/releases/download/webserver%2Fv$version/opentelemetry-webserver-sdk-x64-linux.tgz
```

Comment:
The image seems to be similar to what we already have for apache HTTPd https://github.com/open-telemetry/opentelemetry-operator/blob/main/autoinstrumentation/apache-httpd/Dockerfile#L12 

Is there any reason why we should maintain another one?

Reply:
Ok. I'm going to use the current one then and if anything prevents using the same image in the future, then I'll handle it accordingly.

I'm going to close this PR and update #1853 with a comment that the apache image is used for Nginx, too.